README.md

```diff
@@ -40,13 +40,7 @@ We recommend using MacOS to work with the EDB Docs application.
 
 ### Installation of PDF / Doc Conversion Tools (optional)
```

Comment:
Given this still references PDF conversion, should we add a note about installing Docker too?

Reply:
Good point.